pkg/api-server/inspect_endpoints.go

```diff
@@ -218,6 +225,70 @@ func inspectGatewayDataplanes(
 	}
 }
 
+func inspectGatewayRouteDataplanes(
+	cfg *kuma_cp.Config,
+	builder xds_context.MeshContextBuilder,
+	rm manager.ReadOnlyResourceManager,
+) restful.RouteFunction {
+	return func(request *restful.Request, response *restful.Response) {
+		meshName := request.PathParameter("mesh")
+		gatewayRouteName := request.PathParameter("name")
+
+		meshContext, err := builder.Build(context.Background(), meshName)
```

Comment:
Shouldn't this and others be using `request.Context()`?

Reply:
https://github.com/kumahq/kuma/pull/4108